### ISSUE TITLE
Incorrect Ref base in vcf

### ISSUE THREAD

**Issue by skp38** [freebayes/freebayes#141]
Hello,

I ran the following loop script for single samples:

for i in *sorted.bam; do (freebayes -f MHC.fa -m 30 -q 20 $i> $i.vcf); done

On reviewing the REF base for some POS in the vcf, the REF does not match that position in the reference file (MHC.fa per above). Not all REF are incorrect. It does not appear to be an issue to do with starting position of the reference file versus the vcf.

Any ideas? Help would be greatly appreciated!

**Comment by tseemann** [comment]
I had a similar problem in #140 where I had a reference FASTA file that had Mac and/or Windows line endings. Try running "mac2unix" and "dos2unix" on MHC.fa and see if it solves the problem.

**Comment by ekg** [comment]
This sounds very familiar. Please let us know if this resolves the problem!

**Comment by tseemann** [comment]
@ekg I only filed #140 a couple of days ago, I think it is a bug in freebayes FAI reader.

**Comment by ekg** [comment]
It should probably handle both file formats. It is not set up to detect the
difference. It would be easy to fix.

**Comment by skp38** [comment]
Thanks for your comments. I just ran:
dos2unix MHC.fa
Then I ran
for i in *sorted.bam; do (freebayes -f MHC.fa -m 30 -q 20 $i> $i.vcf); done
on a subset of samples with previously correct and incorrect REFs and checked the output vcfs.
The POS called as snps have changed - there are more now that I have used dos2unix.
Unfortunately some POS REFs are incorrect (albeit different POS than in my initial run).
I tried the above in two ways, one without first indexing MHC.fa and one were I indexed it first, no difference in outcome. 
So it seems to be unresolved so far.
